src/protocolsupport/protocol/packet/middleimpl/serverbound/play/v_pe/LocalPlayerInitialised.java

```diff
@@ -1,31 +0,0 @@
-package protocolsupport.protocol.packet.middleimpl.serverbound.play.v_pe;
-
-import io.netty.buffer.ByteBuf;
-import protocolsupport.listeners.InternalPluginMessageRequest;
-import protocolsupport.protocol.ConnectionImpl;
-import protocolsupport.protocol.packet.middle.ServerBoundMiddlePacket;
-import protocolsupport.protocol.packet.middle.serverbound.play.MiddleCustomPayload;
-import protocolsupport.protocol.packet.middleimpl.ServerBoundPacketData;
-import protocolsupport.protocol.serializer.VarNumberSerializer;
-import protocolsupport.utils.recyclable.RecyclableCollection;
-import protocolsupport.utils.recyclable.RecyclableSingletonList;
-
-public class LocalPlayerInitialised extends ServerBoundMiddlePacket {
```

Comment:
?

Reply:
is it no longer needed?

Reply:
nope, initialization sequence is a bit different now- basically from what i could tell, you wont get a crash as long as PLAYER_SPAWN is sent when the chunk the player is in has already been sent. and dimension switch waits on the dim-switch-ack